src/services/AssetService.ts

```diff
@@ -126,15 +131,28 @@ export class Assets {
         }
 
         return axios.get(url).then((response: AxiosResponse) => {
-            for (const result of response.data) {
-                if (this.matches(term, result.name)) {
-                    matches.push({
-                        name: result.name,
-                        id: result[this.idProperty],
-                        type: this.assetType
-                    });
+            // Only attempt to match if response contains a list of externally-fetched assets
```

Comment:
This is my answer to handling the single-asset response our mocked `environment` endpoint is currently returning, what do you think? This change was necessary after updating `SetContactAttribForm` to pass the `EnvironmentAsset` service to `SelectSearch`.

Reply:
@ericnewcomer this one's ready for another review 👍

Agree that having duplicate `typeConfig`s and a language map aren't ideal, happy to address these down the road.